README.md

```diff
@@ -1,63 +1,73 @@
+
 # elabforms
-A set of tools to create and manage standardized forms for eLabFTW
+
+A set of tools to create and manage standardized forms for eLabFTW.
+
+---
 
 ## Create a virtual environment
+
 ```bash
 python3 -m venv myenv
 source myenv/bin/activate
-```
+````
+
+---
 
 ## Installation
-you will need temaplates_part files to generate the final template.
-these files are located in the `elabforms_INTProjects` and `elabforms_BIDSMetadata` repositories. int NIT PRIVATE git repositories.
-Make sure you have `git` and `pip` installed. You can download and unzip the following repositories:
-- git@github.com:INT-NIT/elabforms_INTProjects.git
-- git@github.com:INT-NIT/elabforms_BIDSMetadata.git
 
-Or you can clone the repositories using `git`.
+You will need `template_part` files to generate the final template.
+These files are located in the `elabforms_INTProjects` and `elabforms_BIDSMetadata` repositories — private Git repositories at NIT.
```

Comment:
you still mention the INT resources (private repo & co)... this should not be here!
I will correct it myself so that we can merge quickly...